doc/1/exceptions/internal-exception/index.md

```diff
@@ -0,0 +1,13 @@
+---
+code: false
+type: page
+title: InternalException
+description: InternalException
+order: 0
+---
+
+# InternalException
+
+Inherits from the [KuzzleException](/sdk/csharp/1/exceptions/kuzzle-exception) class.
+
+The `InternalException` class is thrown when submitting a malformated query to the [Kuzzle.QueryAsync](/sdk/csharp/1/core-classes/kuzzle/query-async) method.
```

Comment:
```suggestion
The `InternalException` exception is thrown when submitting a malformed query to the [Kuzzle.QueryAsync](/sdk/csharp/1/core-classes/kuzzle/query-async) method.
```

Reply:
(nitpicking) Also, I find this description a bit too specific. To me, internal errors are thrown on an unexpected SDK behavior.

Reply:
Yes you're right!